x/leverage/keeper/keeper_test.go

```diff
@@ -126,7 +126,7 @@ func (s *IntegrationTestSuite) setupAccount(denom string, mintAmount, supplyAmou
 
```

Comment:
These new return fields can be checked to improve our test suite in the future, but I want to hold off on that until I refactor `x/leverage/keeper` tests.

For now, test behavior is unchanged.